Prog/Make_all_profiles.sas

```diff
@@ -38,16 +38,16 @@
 %include "L:\SAS\Inc\StdLocal.sas";
 
 ** Define libraries **;
-%DCData_lib( Profiles, local=y )
-%DCData_lib( Ncdb, local=y )
-%DCData_lib( ACS, local=y )
-%DCData_lib( Vital, local=y )
-%DCData_lib( Hmda, local=y )
-%DCData_lib( Police, local=y )
-%DCData_lib( RealProp, local=y )
-%DCData_lib( TANF, local=y )
-%DCData_lib( ROD, local=y )
-%DCData_lib( Schools, local=y )
+%DCData_lib( Profiles )
+%DCData_lib( Ncdb, local=n )
```

Comment:
I changed this to default to the remote libraries for all profile data. _Profiles_ library should **not** default to remote library so that macro changes in the local repo will be used during testing.